client/my-sites/stats/stats-notices/index.tsx

```diff
@@ -104,8 +104,7 @@ const NewStatsNotices = ( { siteId, isOdysseyStats, statsPurchaseSuccess }: Stat
 			return hasReachedPaywallMonthlyViews( state, siteId );
 		} ) &&
 		! supportCommercialUse &&
-		isSiteJetpackNotAtomic &&
-		! isVip;
```

Comment:
Isn't this still needed?

Reply:
I was trying to leave the `isVip` check on the [notice definition where it was used](https://github.com/Automattic/wp-calypso/blob/127a7d648d47036cf2766cda0b01d271caa52a3c/client/my-sites/stats/stats-notices/all-notice-definitions.ts#L48), which makes the `shouldShowPaywallNotice` focus on its meaning.

Reply:
That makes sense, thanks.